tests/bluetooth/bsim_bt/bsim_test_notify/src/gatt_server_test.c

```diff
@@ -210,10 +267,28 @@ static void test_main(void)
 
 static const struct bst_test_instance test_gatt_server[] = {
 	{
-		.test_id = "gatt_server",
+		.test_id = "gatt_server_none",
+		.test_post_init_f = test_init,
+		.test_tick_f = test_tick,
+		.test_main_f = test_main_none,
+	},
+	{
+		.test_id = "gatt_server_unenhanced",
+		.test_post_init_f = test_init,
+		.test_tick_f = test_tick,
+		.test_main_f = test_main_unenhanced,
+	},
+	{
+		.test_id = "gatt_server_enhanced",
+		.test_post_init_f = test_init,
+		.test_tick_f = test_tick,
+		.test_main_f = test_main_enhanced,
+	},
+	{
+		.test_id = "gatt_server_mixed",
 		.test_post_init_f = test_init,
 		.test_tick_f = test_tick,
-		.test_main_f = test_main,
+		.test_main_f = test_main_mixed,
```

Comment:
Btw, it is also possible to provide arguments when starting a test using the `-argstest` argument. It will basically provide the same as this, so there's no need to modify anything, but it could be a way of just having a single main function and a single test_id, by providing the `chan_opt` as a argument to the test.

Reply:
What we have done some of the other places, is that we have a `_<name>.sh` file (which isn't picked up by CI) to run all the tests locally like here: https://github.com/zephyrproject-rtos/zephyr/blob/d4dfb68ee5b083c8db87b9bd5db37041435726b5/tests/bluetooth/bsim_bt/bsim_test_audio/test_scripts/_cap.sh

Reply:
alright I was wondering too if it was because of a limitation of CI. I hope we can work out something better than these scripts for twister v2.

Reply:
Yes, that would be a nice improvement over the current setup, but it was postponed until twister v2. I do not what the timeline on that is.